.gitmodules

```diff
@@ -1,15 +1,11 @@
 [submodule "FV3"]
   path = FV3
-  url = https://github.com/NOAA-EMC/fv3atm
+  url = https://github.com/lisa-bengtsson/fv3atm
   branch = develop
```

Comment:
The branch should also point to your prog_closure branch instead of develop here.